interface.go

```diff
@@ -209,10 +188,11 @@ func NewInterface(ctx context.Context, c *InterfaceConfig) (*Interface, error) {
 		l: c.l,
 	}
 
-	if myVpnAddr.Is4() {
-		addr := myVpnNet.Masked().Addr().As4()
-		binary.BigEndian.PutUint32(addr[:], binary.BigEndian.Uint32(addr[:])|^binary.BigEndian.Uint32(certificate.Details.Ips[0].Mask))
-		ifce.myBroadcastAddr = netip.AddrFrom4(addr)
+	if ifce.myVpnNet.Addr().Is4() {
+		//TODO:
```

Comment:
I can bring this back but in the next iteration we will have 0 to N ipv4 networks in certificates and will need to assemble a cidr tree to efficiently match the broadcast address of any of the networks. This was originally included to silence annoying logs related to network device discovery and continuing to keep it will reduce performance.